src/components/ec-radio-btn/__snapshots__/ec-radio-btn.spec.ts.snap

```diff
@@ -0,0 +1,952 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EcRadioBtn focus should be checked and focused 1`] = `
+<body>
+  <div>
+    <div
+      data-v-app=""
+    />
+  </div>
+  <div>
+    <div
+      data-v-app=""
+    >
+      <div
+        class="ec-radio-btn"
+        data-test="ec-radio-btn ec-radio-btn"
+      >
+        <input
+          aria-describedby="ec-radio-btn-1-error ec-radio-btn-1-label ec-radio-btn-1-description"
+          class="ec-radio-btn__input"
+          data-test="ec-radio-btn__input"
+          id="ec-radio-btn-1"
```

Comment:
we need to provide the same id attribute for label and description

id="ec-radio-btn-1-label"
id="ec-radio-btn-1-description"